app/views/contributionsRedirect.scala.html

```diff
@@ -0,0 +1,50 @@
+@import assets.AssetsResolver
+@()(implicit assets: AssetsResolver)
+
+
+<!DOCTYPE html>
```

Comment:
Why do the redirect client side rather than with a 301?

Reply:
This was the quickest route to fixing the problem.  The geo location isn't currently available server-side.  Ideally this can be removed once we have a separate email for US and UK paypal failures

Reply:
Probably worth putting a comment in the controller or the route to remind us of that.